packages/frontend/src/components/accounts/ledger/SignInLedgerWrapper.js

```diff
@@ -39,7 +41,8 @@ export function SignInLedgerWrapper(props) {
 
     const [accountId, setAccountId] = useState('');
     const [loader, setLoader] = useState(false);
-    const [confirmedPath, setConfirmedPath] = useState(1);
+    const [confirmedPath, setConfirmedPath] = useState(10);
```

Comment:
Was the change to `10` intentional?

Reply:
Thanks for catching!! Leftover from testing. Definitely not intentional.